config/routes.rb

```diff
@@ -6,6 +6,13 @@
     resources :reviews, except: [:show, :index, :destroy]
   end
 
+  resources :reviews, only: [] do
+    resources :votes, only: [:destroy]
+    get '/up-vote', to: 'votes#up_vote', as: :up_vote
+    get '/down-vote', to: 'votes#down_vote', as: :down_vote
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.